bread/views/browse.py

```diff
@@ -248,15 +231,24 @@ def get_paginate_by(self, queryset):
     def get_queryset(self):
         """Prefetch related tables to speed up queries. Also order result by get-parameters."""
         qs = super().get_queryset()
-        if (
-            self.search_backend
-            and self.search_backend.query_parameter in self.request.GET
-        ):
-            qs = queryset_from_fields.get_field_queryset(
-                [*self.model._meta.fields, *self.model._meta.many_to_many],
-                self.request.GET.getlist(self.search_backend.query_parameter),
-            )
-            qs = self.model.objects.filter(qs)
+        if self.search_urlparameter and self.search_urlparameter in self.request.GET:
+            searchquery = self.request.GET[self.search_urlparameter].strip()
+            if searchquery.startswith("="):
+                try:
+                    qs = apply_search(qs, searchquery[1:])
+                except DjangoQLError as e:
+                    messages.error(
+                        self.request,
+                        _("Bad filter string '%s': '%s'") % (searchquery, e),
+                    )
+
+            else:
+                qs = self.model.objects.filter(
+                    queryset_from_fields.get_field_queryset(
+                        [*self.model._meta.fields, *self.model._meta.many_to_many],
+                        searchquery,
+                    )
+                )
```

Comment:
Added this: When starting with the = sign we can use djangoql for the search string.

Reply:
If DjangoQL tends to work better in most cases (like complicated search), why don't we replace the old one with entirely DjangoQL then?

Reply:
Hmm... That might actually be a good idea. I already looked at it briefly in the past, but because we still need to make distinctions between different field types I thought it would not add that much value (e.g. we have to generate ```'id = 3'``` and not ```'id = "3"'```).
However, it might still make sense to simplify ```get_field_queryset``` maybe?